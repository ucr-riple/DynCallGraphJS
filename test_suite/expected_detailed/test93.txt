{
    'global (test93.js:12:9:12:28)': [
        'anon (test93.js:1:25:11:2)'
    ],
    'global (test93.js:13:1:13:14)': [
        'anon (test93.js:7:16:9:6)'
    ]
}