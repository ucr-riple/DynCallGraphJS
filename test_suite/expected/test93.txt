{ '(test93.js:12:9:12:28)':
   [ '(test93.js:1:25:11:2)' ],
  '(test93.js:13:1:13:14)':
   [ '(test93.js:7:16:9:6)' ] }